.github/workflows/build.yml

```diff
@@ -0,0 +1,18 @@
+name: build the package
+on:
+  pull_request:
+jobs:
+  run:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19.x
+      - name: Checkout code
```

Comment:
I think we can remove the name field on all these self-explanatory steps.

Reply:
removed